gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/SerializationTest.java

```diff
@@ -133,20 +133,21 @@
 
   @Test
   public void testServiceOptions() throws Exception {
-    DatastoreOptions options = DatastoreOptions.builder()
-        .authCredentials(AuthCredentials.createForAppEngine())
-        .normalizeDataset(false)
-        .projectId("ds1")
-        .build();
+    DatastoreOptions options =
+        DatastoreOptions.builder()
+            .normalizeDataset(false)
+            .projectId("ds1")
+            .build();
     DatastoreOptions serializedCopy = serializeAndDeserialize(options);
     assertEquals(options, serializedCopy);
 
-    options = options.toBuilder()
-        .namespace("ns1")
-        .retryParams(RetryParams.getDefaultInstance())
-        .authCredentials(AuthCredentials.noCredentials())
-        .force(true)
-        .build();
+    options =
```

Comment:
They were preferred, not necessary.  I reverted that style change here and in storage's SerializationTest.